app/src/main/res/values/strings.xml

```diff
@@ -26,4 +26,15 @@
     <string name="theme_night">Night</string>
     <string name="theme_battery">Follow battery saver</string>
     <string name="theme">Theme</string>
+    <string name="shortcut">Shortcut</string>
+    <string name="icon_picker">Icon Picker</string>
+    <string name="create_or_append_to_an_existing_shortcut_rather_than_apply">Create or append to an existing shortcut rather than applying it</string>
+    <string name="edit_shortcuts">Edit shortcuts</string>
+    <string name="grant_secure_setting">Grant secure setting</string>
+    <string name="append">Append</string>
+    <string name="disable">Disable</string>
+    <string name="enable">Enable</string>
+    <string name="disabled">Disabled</string>
+    <string name="delete_action">Delete Action</string>
+    <string name="cancel">Cancel</string>
```

Comment:
You need to be more elaborative when you name strings, because it can be confusing in some languages. For example, you could use `action_append` or `status_disabled`, etc.

Reply:
You're right again.